notifications/drf_views.py

```diff
@@ -76,6 +79,11 @@ def get_serializer_class(self):
         # return UnauthenticatedSurgeAlertSerializer
         return SurgeAlertSerializer
 
+    def get_queryset(self):
+        base_queryset = self.queryset
+        my_queryset = base_queryset.filter(molnix_id__isnull=False).exclude(molnix_tags__name="NO_GO")
+        return my_queryset
```

Comment:
We should always use super().get_queryset() instead of self.queryset as self.queryset might not be defined.
```suggestion
    def get_queryset(self):
        queryset = super().get_queryset()
        return queryset.filter(molnix_id__isnull=False).exclude(molnix_tags__name="NO_GO")
```

For more detail please check: https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py#L52-L77